string attribute_name
float32 score
